numpy/__init__.pyi

```diff
@@ -4492,13 +4482,12 @@ class poly1d:
     @coefficients.setter
     def coefficients(self, value: NDArray[Any]) -> None: ...
 
-    __hash__: ClassVar[None]  # type: ignore
+    __hash__: ClassVar[None]  # type: ignore[assignment]  # pyright: ignore[reportIncompatibleMethodOverride]
```

Comment:
I'm confused by the comment `ignore[reportIncompatibleMethodOverride]`.  Is it a directive to pyright?

Reply:
Yea, pyright uses these different type-ignore comments. For those using vscode with pylance it can help to filter out false positives like these.